### PR TITLE
Added checks for friends on triggerbot

### PR DESCRIPTION
**Pull request by SwiftHD**
=)

### CODE REVIEW

**Comment by SurvirvorHRG** [comment]
the code is kinda weird

**Comment by SwiftHD** [comment]
but it kinda works

File: Horion/Module/Modules/TriggerBot.cpp [hunk 1]
```diff
@@ -16,20 +17,37 @@ const char* TriggerBot::getModuleName()
...
+			return;
+
+		else {
 		if (localPlayer->getEntityTypeId() == target->getEntityTypeId())
 		{
 			localPlayer->swingArm();
 			gm->attack(target);
 		}
-		Odelay = 0;	
+			Odelay = 0;
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp]: Revert this line

File: Horion/Module/Modules/TriggerBot.cpp [hunk 2]
```diff
@@ -16,20 +17,37 @@ const char* TriggerBot::getModuleName()
 {
 	return ("TriggerBot");
 }
-
+static std::vector <C_Entity*> targetList0;
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp]: not needed

File: Horion/Module/Modules/TriggerBot.cpp [hunk 3]
```diff
@@ -16,20 +17,37 @@ const char* TriggerBot::getModuleName()
...
 void TriggerBot::onTick(C_GameMode* gm)
 {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 	C_Entity* target = g_Data.getClientInstance()->getPointerStruct()->entityPtr;
 	Odelay++;
 	if (target != 0 && Odelay >= delay)
 	{
+		if (target == g_Data.getLocalPlayer()) // Skip Local player
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp]: `target == localPlayer ` would be a better choice here

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by SwiftHD** [comment]
done changes. should of updated

File: Horion/Module/Modules/TriggerBot.cpp [hunk 4]
```diff
@@ -17,15 +17,14 @@ const char* TriggerBot::getModuleName()
...
-static std::vector <C_Entity*> targetList0;
 void TriggerBot::onTick(C_GameMode* gm)
 {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 	C_Entity* target = g_Data.getClientInstance()->getPointerStruct()->entityPtr;
 	Odelay++;
 	if (target != 0 && Odelay >= delay)
 	{
-		if (target == g_Data.getLocalPlayer()) // Skip Local player
+		if (target == localPlayer()) // Skip Local player
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp:27]: remove the `()`

**Review by ghost — CHANGES_REQUESTED** [review]
Please compile before pushing changes

**Comment by ghost** [comment]
he mad

**Pull request closed without merging**